tests/cover/test_datetimes.py

```diff
@@ -45,11 +44,11 @@ def test_can_find_negative_delta():
 
 
 def test_can_find_on_the_second():
-    minimal(timedeltas(), lambda x: x.seconds == 0)
+    timedeltas().filter(lambda x: x.seconds == 0).example()
```

Comment:
Because Travis was slow and several of them timed out... but I've changed enough to make that a separate pull I guess.